### ISSUE TITLE
Report "Pageviews" as well as "Actions" for all custom dimension values 

### ISSUE THREAD

**Issue by mattab** [matomo-org/matomo#16160]
it would be very valuable (and more consistent) to also report on Pageviews along with Actions metrics in custom dimension reports, for each value.

**Comment by AkshayRao27** [comment]
I agree. I ended up implementing a bunch of Custom Reports literally just because of this one problem. The "Actions" metric is basically useless for our situation because it's such a mix of various different metrics. The sidebar in our Matomo is unnecessarily cluttered because of this.
